src/lib/style-formatting/index.ts

```diff
@@ -160,6 +160,17 @@ function handleDeepObject(value: any, parameter: ParameterObject) {
 // Explode is handled on its own, because style-serializer doesn't return what we expect for proper
 // HAR output.
 function handleExplode(value: any, parameter: ParameterObject) {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Added this due to the following error:

```
Property 'type' does not exist on type 'ReferenceObject | SchemaObject'.
  Property 'type' does not exist on type 'ReferenceObject'.ts(2339)
```

parameter.schema is coming up as this:

```ts
interface ParameterBaseObject {
    description?: string;
    required?: boolean;
    deprecated?: boolean;
    allowEmptyValue?: boolean;
    style?: string;
    explode?: boolean;
    allowReserved?: boolean;
    schema?: ReferenceObject | SchemaObject;
    example?: any;
    examples?: {
        [media: string]: ReferenceObject | ExampleObject;
    };
    content?: {
        [media: string]: MediaTypeObject;
    };
}
```

And SchemaObject is this, which seems correct?

```ts
type SchemaObject = ArraySchemaObject | NonArraySchemaObject;
interface ArraySchemaObject extends BaseSchemaObject {
    type: ArraySchemaObjectType;
    items: ReferenceObject | SchemaObject;
}
interface NonArraySchemaObject extends BaseSchemaObject {
    type?: NonArraySchemaObjectType;
}
```

Reply:
So @erunion and I have seen this before. `schema?: ReferenceObject | SchemaObject;` isn't just a `SchemaObject`, it might be a `ReferenceObject`, which doesn't have a `type`.

@erunion put a bunch of work into removing `ReferenceObject`s (since we're de-referencing before it hits this code), but I can't remember where. He's the best one to help with this, but if it isn't easy ts-ignore with a comment explaining the above is good enough for me.

Reply:
I fixed it with some type coercion, which seems to do the trick but i'm definitely still a TS noob and there may be a much better way to do this: ce0d505fec16bedd3383ad6d20edd97a675d0a05